CMakeLists.txt

```diff
@@ -47,6 +47,8 @@ lxqt_translate_ts(QM_FILES
     ${UIS}
     INSTALL_DIR
     "${LXQT_TRANSLATIONS_DIR}/${PROJECT_NAME}"
+    PULL_TRANSLATIONS ${PULL_TRANSLATIONS}
```

Comment:
@palinek They are options. Check it [here](https://github.com/lxde/liblxqt/blob/lp-ext_translations/cmake/modules/LXQtTranslateTs.cmake#L93)